developers/blobstreamx-offchain.md

```diff
@@ -1,18 +1,18 @@
 ---
-description: Learn how to integrate your L2's offchain logic with Blobstream
+description: Learn how to integrate your L2's offchain logic with Blobstream X
 ---
 
-# Integrate with Blobstream client
+# Integrate with Blobstream X client
 
-## Blobstream demo rollup
+## BlobstreamX demo rollup
 
-Rollups can use the Blobstream for DA by posting their data to Celestia and then
+Rollups can use the Blobstream X for DA by posting their data to Celestia and then
 proving that it was posted on Ethereum. This is done identically to how any
-rollup or user would post data to Celestia, and then the validators sign over
-additional commitments that are relayed to Ethereum via a light client relay
-(aka Blobstream!). This demo will outline (note outline is not an
+rollup or user would post data to Celestia, and then a zk-proof of the Celestia
```

Comment:
nit: "to Celestia. Then, a zero-knowledge proof (ZKP) that Celestia validators have come to consensus on Celestia block headers is generated, and subsequently relayed to Ethereum to the Blobstream X smart contract. A demo rollup wanting to use Blobstream X ..."

Reply:
Can you precise more what to change? 

maybe use the`suggestion`:

```suggestion
rollup or user would post data to Celestia, and then a zk-proof of the Celestia
```

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/5c84be33e8332e294d78b9f74bba0ce1b6a95736 note: i did not put the abbreviation "ZKP" in the doc, because it isn't used in the page